app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/XmlWriter.scala

```diff
@@ -34,15 +35,26 @@ trait XmlWriter[A] {
   }
 
   protected def element(elementName: String, elementValue: String, attribute: Option[Attribute] = None ): Elem = {
+    elementContruct(elementName, attribute, Text(elementValue))
+  }
+
+  protected def maybeDateTimeElement(elementName: String, maybeElementValue: Option[String], attribute: Option[Attribute] = None): Option[Node] = {
+    maybeElementValue.filter(_.nonEmpty).map { value =>
+      val formatAttrib = Attribute.apply(pre = "", key = "formatCode", "304", scala.xml.Null)
+      val dtElement = Elem.apply("p1", "DateTimeString", formatAttrib, scala.xml.TopScope, true, Text(value))
```

Comment:
do we actually need the namespace prefix here?  if so, should it be a parameter?

Reply:
Without the prefix schema validation fails with: `Invalid content was found starting with element '{"urn:wco:datamodel:WCO:DEC-DMS:2":DateTimeString}'. One of 
'{"urn:wco:datamodel:WCO:Declaration_DS:DMS:2":DateTimeString}' is expected.`

Reply:
Why make it a parameter though? Are you thinking about extending the method to be able to write other DateTimeString elements?

Reply:
just that it has to match the namespace id from `<Declaration ... xmlns:p1="urn:wco:datamodel:WCO:Declaration_DS:DMS:2" ...>` which might (eventually) be dynamically generated too

Reply:
Leave for now, look again if thinks change down the line.

Reply:
P1 (or whatever else we call it) is needed as it refers to namespace xmlns:p1="urn:wco:datamodel:WCO:Declaration_DS:DMS:2"

where the type is defined.